ntbs-service/Middleware/AuditGetRequestMiddleware.cs

```diff
@@ -49,6 +49,13 @@ public async Task InvokeAsync(HttpContext context, IAuditService auditService)
                     await auditService.AuditNotificationReadAsync(id, NotificationAuditType.Full, userName);
                 };
             }
+
+            if (response.StatusCode == StatusCodes.Status200OK
+                && pathArray.Contains("search")
```

Comment:
I think we need to be more specific here - "search" could well be used in other endpoints. I think we want to check that "search" is the only part of the path.

Reply:
On a related note, it would be nice if we didn't include audits of searches which fail validation... but this isn't specified by the ACs and would be slightly hacky so probably isn't worth it.

Reply:
Path array is an array of the path split on '/', so if there was a new page e.g. ntbs.com/SearchUsers it wouldn't match this. But I agree a call to Single() here would make it safer.

Reply:
No, but there could be a page like `/SomeOtherObject/Search`